tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Steps/UiValidationSteps.cs

```diff
@@ -70,12 +70,26 @@ public void ThenTheElementContainsALinkTo(string dataId, string link)
             linkElement.Should().NotBeNull($"an element with link {link} should be found");
         }
 
+        [Then(@"the user clicks the element with Data ID ([^\s]+)")]
+        public void ThenTheUserClicksTheRequest_Account_Link(string dataId)
```

Comment:
Given instead of Then?

Also

GivenTheUserClicksElementWithDataId would be a better name